packages/ui/src/components/TagInput/index.tsx

```diff
@@ -220,48 +276,139 @@ export const TagInput = ({
     try {
       dispatchOnChange(newTagInput)
       setStatus({ [newTagInput.length - 1]: STATUS.IDLE })
-    } catch (error) {
-      onChangeError?.(error as Error)
+    } catch (err) {
       setTagInput(tagInputState)
     }
   }
 
+  const clearAll = () => {
+    setInput('')
+    setTagInput([])
+    dispatchOnChange([])
+  }
+
+  const helperSentiment = useMemo(() => {
+    if (error) {
+      return 'danger'
+    }
+
+    if (success) {
+      return 'success'
+    }
+
+    return 'neutral'
+  }, [error, success])
+
+  const computedClearable = clearable && !!tagInputState.length
+
   return (
-    <TagInputContainer
-      onClick={handleContainerClick}
-      variant={variant}
-      className={className}
-      data-testid={dataTestId}
-    >
-      {tagInputState.map(tag => (
-        <Tag
-          sentiment="neutral"
-          disabled={disabled}
-          key={tag.index}
-          isLoading={status[tag.index] === STATUS.LOADING}
-          onClose={e => {
-            e.stopPropagation()
-            deleteTag(tag.index)
-          }}
+    <Stack gap="0.5" className={className}>
+      <Stack direction="row" gap="1" alignItems="center">
+        <Stack direction="row" gap="0.5" alignItems="start">
+          <Text
+            as="label"
+            variant="bodyStrong"
+            sentiment="neutral"
+            htmlFor={id ?? localId}
+          >
+            {label}
+          </Text>
+          {required ? <Icon name="asterisk" color="danger" size={8} /> : null}
+        </Stack>
+        {labelDescription ?? null}
+      </Stack>
+
```

Comment:
This is a bit fucked up but we need to encapsulate tooltip into a div so it inherit from display block in this case, I had the same issue with other components if we don't do it the width is broken when setting a tooltip

<img width="572" alt="Screenshot 2024-02-07 at 18 00 43" src="https://github.com/scaleway/ultraviolet/assets/15812968/2d906077-b137-4f9a-b1cb-a5e13329e11a">